### DISCUSSION TITLE
"Clickable Action" - A Component to Showcase a Clickable Action

### DISCUSSION THREAD

**Discussion by andrewballantyne** [patternfly/patternfly#5485] [Feature requests]
**Is this a new component or an extension of an existing one?**

This would be a new component.

We styled it off of the Editable Label Group (eg https://www.patternfly.org/v4/components/label-group#editable-labels).

**Describe the feature**

A contextual clickable action to prompt the user there is more content there. We needed a low footprint way to give a clickable notion to `Starting...` text for when the resource was spinning up and going through K8s setup. This granted them more options to view how that was going, with and without an blocking error state. 

Normal running:
![image](https://user-images.githubusercontent.com/8126518/214142081-a95f0c85-461e-42a8-84ee-27210304208f.png)

Hidden error -- click for more info, had tooltip for short text on the error description:
![image](https://user-images.githubusercontent.com/8126518/214141415-b01c4ead-a06d-430a-9c14-378910c7eee5.png)

The popup with the fully detailed k8s (in this case) error:
![image](https://user-images.githubusercontent.com/8126518/214141498-95583141-e2f2-4541-87ff-7000af50b551.png)

**Are there visuals for this feature? If applicable, please include examples for each state and for varying widths**

We followed the styles of the Editable Label Group. Underline, cursor finger. We added an error icon with tooltip when it was applicable -- not sure if that's needed part of this component, it could just be the content.

**Any other information?**

Working with the UX team on RHODS we came up with this action so we could bring attention to the user when there was additional contextual information for them to get at if they wanted to. If there was an issue, we combined it with an error icon and tooltip hover to help draw more attention.

**Comment by andrewballantyne** [comment]
cc @kywalker-rh

**Comment by mcarrano** [comment]
Transferring for more discussion
